src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.DotNet.XUnitExtensions;
```

Comment:
nit: we usually add non-System.* 'using' statements after all the System.* using statements.

Reply:
updated. I used this order as that preserves alphabetical order.

Reply:
>updated. I used this order as that preserves alphabetical order.

yes.  And most things are alphabetical.  But the System.* using statements are collectively listed first before any custom ones like "Microsoft.*".  So, the rule has this exception.